### PR TITLE
DFCC-542 Draft graph

### PR DESCRIPTION
**Pull request by lazcool**

### CODE REVIEW

File: DFC.ServiceTaxonomy.Editor/DFC.ServiceTaxonomy.Editor.csproj [hunk 1]
```diff
@@ -33,15 +33,20 @@
 
   <ItemGroup>
     <None Include="..\.editorconfig" Link=".editorconfig" />
+    <Content Remove="appsettings.Development_template.json" />
```
Comment (tomtollafield) [inline DFC.ServiceTaxonomy.Editor/DFC.ServiceTaxonomy.Editor.csproj:36]: Did you mean to leave this in?

File: DFC.ServiceTaxonomy.Editor/Startup.cs [hunk 2]
```diff
@@ -21,6 +23,14 @@ public void ConfigureServices(IServiceCollection services)
                 options.InstrumentationKey = Configuration["APPINSIGHTS_INSTRUMENTATIONKEY"]);
 
             services.AddOrchardCms();
+
+            services.AddEventGridPublishing(Configuration);
+
+            //todo: do this in each library??? if so, make sure it doesn't add services or config twice
+            services.AddGraphCluster(options =>
```
Comment (tomtollafield) [inline DFC.ServiceTaxonomy.Editor/Startup.cs:30]: We might want to do a string null check on the section to make sure we're not binding to null. If you already check later on though, ignore this comment.

File: DFC.ServiceTaxonomy.Editor/DFC.ServiceTaxonomy.Editor.csproj (hunk 1 above)
Comment (lazcool) [reply DFC.ServiceTaxonomy.Editor/DFC.ServiceTaxonomy.Editor.csproj:36]: Yes, without it the file gets copied to the bin folder.

File: DFC.ServiceTaxonomy.Editor/Startup.cs (hunk 2 above)
Comment (lazcool) [reply DFC.ServiceTaxonomy.Editor/Startup.cs:30]: The MSDN states that `GetSection()` returns `null` if the section is missing, except it doesn't, it returns a valid (empty) section instead 😄.
So instead, I've added checks for missing config items in `GraphClusterBuilder` and it now throws a `ConfigurationErrorsException` is there are no endpoints or replica sets configured.

**Pull request merged**